test_conformance/vulkan/test_vulkan_api_consistency_for_1dimages.cpp

```diff
@@ -158,47 +158,6 @@ int test_consistency_external_for_1dimage(cl_device_id deviceID,
     test_error(errNum, "Unable to create Image with Properties");
     image.reset();
 
-    // Passing properties, image_desc and image_format all as NULL
-    image = clCreateImageWithProperties(context, NULL, CL_MEM_READ_WRITE, NULL,
```

Comment:
I think this should be retained for the same reason I mentioned above.

Reply:
This test case does not make sense for this extension, there is no information being passed that assosciates clCreateImageWithProperties and cl_khr_external_memory. 

If we believe this is important coverage, it would be a better fit in api or basic.

Reply:
I prefer we retain the coverage here until we add it to more appropriate place rather than removing it right away.
We can either add the same to more appropriate tests or move these tests to more appropriate place in a subsequent PR.

Reply:
Sure, we can leave in the two valid negative tests.  I made an issue to cover these https://github.com/KhronosGroup/OpenCL-CTS/issues/2139